### ISSUE TITLE
[Security] Use TPM

### ISSUE THREAD

**Issue by zjkmxy** [zjkmxy/ndn-workspace-solid#19]
Currently private keys are stored in plain text in the storage. This could be insecure.
We should use TPM to avoid this. Or at least provide the option to use TPM.

https://github.com/zjkmxy/ndn-workspace-solid/blob/3e46cb7757ff9fcdaeabd339c16a4212d175e01a/src/backend/models/profiles.ts#L7-L13
https://github.com/zjkmxy/ndn-workspace-solid/blob/3e46cb7757ff9fcdaeabd339c16a4212d175e01a/src/backend/models/connections.ts#L7-L13

**Comment by zjkmxy** [comment]
https://github.com/zjkmxy/ndn-workspace-solid/blob/3e46cb7757ff9fcdaeabd339c16a4212d175e01a/src/components/connect/ndn-testbed.tsx#L88-L90
https://github.com/zjkmxy/ndn-workspace-solid/blob/3e46cb7757ff9fcdaeabd339c16a4212d175e01a/src/components/connect/ndn-testbed.tsx#L107-L109
https://github.com/zjkmxy/ndn-workspace-solid/blob/3e46cb7757ff9fcdaeabd339c16a4212d175e01a/src/components/workspace/hackathon.tsx#L46-L48

**Comment by yoursunny** [comment]
NDNts has `KeyChain` type that stores private keys in IndexedDB.
These keys are generated as `extractable: false`, which is as secure as you can get in a browser without additional plugins or external smartcards.

**Comment by zjkmxy** [comment]
Yes. I'm aware of that. This issue is merely some coding on my side.
